### PR TITLE
support for longformer (new 2023 request)

### PR DESCRIPTION
**Pull request by jtfields**
I'm new to creating pull requests so please give me feedback if anything here is not correct.  I created the pull request comparing the Multimodal-Toolkit with the fork that I created.  I should also note that I had to downgrade transformers in my Colab notebook file to get the code to run (!pip install transformers==4.28.0).  Thanks again for all the help to get this to work and let me know if you have any questions.

### CODE REVIEW

File: multimodal_transformers/model/tabular_combiner.py
Comment (akashsaravanan-georgian) [inline multimodal_transformers/model/tabular_combiner.py:1]: Could we remove all the comments here? I think these might be leftover from your tests and may confuse future contributors.

File: notebooks/text_w_tabular_classification.ipynb
Comment (akashsaravanan-georgian) [inline notebooks/text_w_tabular_classification.ipynb:1]: Could you restore this notebook? We don't want this to be removed.

File: notebooks/Longformer_text_w_tabular_classification_051823.ipynb
Comment (akashsaravanan-georgian) [inline notebooks/Longformer_text_w_tabular_classification_051823.ipynb:1]: Could you rename this to remove the numbers at the end? Just use something like `longformer_text_w_tabular_classification.ipynb`

File: setup.py
Comment (akashsaravanan-georgian) [inline setup.py:1]: Could we revert the changes made here?

**Review by akashsaravanan-georgian — CHANGES_REQUESTED** [review]
Hey @jtfields, thank you for all the help! I went through your code and suggested a few very minor changes. Would appreciate it if you could make those changes. I should be able to merge this into the repo after that!

File: multimodal_transformers/model/layer_utils.py
[the remark this answers is not in the document]
Comment (jtfields) [reply multimodal_transformers/model/layer_utils.py:1]: I removed the commented line and the 3 new lines.

File: notebooks/Longformer_text_w_tabular_classification_051823.ipynb
Comment (jtfields) [reply notebooks/Longformer_text_w_tabular_classification_051823.ipynb:1]: I renamed the file.  You will need to compare the changes in the notebook with the original.  For example, I added a specific transformers version and used the metrics workaround for the bug in order to make the code run.

File: setup.py
Comment (jtfields) [reply setup.py:1]: I removed the changes from setup.py

**Review by akashsaravanan-georgian — APPROVED** [review]

**Comment by akashsaravanan-georgian** [comment]
Great work @jtfields!! I'm merging the PR now. Out of curiosity, do you know why you had to downgrade transformers to get longformers working in the notebook?

**Pull request merged**